FF1Blazorizer/Tabs/EnemiesTab.razor

```diff
@@ -9,15 +9,17 @@
 		<div class="col1">
 			<h4>Formations</h4>
 			<EnumDropDown UpdateToolTip="@UpdateToolTipID" Id="formationShuffleModeDropDown" TItem="FormationShuffleMode" IsEnabled="@(!Flags.RandomizeFormationEnemizer)" @bind-Value="Flags.FormationShuffleMode">Rarity:</EnumDropDown>
-			<EnumDropDown UpdateToolTip="@UpdateToolTipID" Id="runnabilityDropDown" TItem="Runnability" @bind-Value="Flags.Runnability">Unrunnable Formations:</EnumDropDown>
-			<TriStateCheckBox UpdateToolTip="@UpdateToolTipID" Id="unrunnablesStrikeFirstAndSurpriseCheckBox" @bind-Value="Flags.UnrunnablesStrikeFirstAndSurprise">Unrunnable First Strike &amp; Surprise</TriStateCheckBox>
+
+            <TriStateCheckBox UpdateToolTip="@UpdateToolTipID" Id="runnabilityShuffleCheckbox" @bind-Value="Flags.UnrunnableShuffle">Shuffle Unrunnable Encounters</TriStateCheckBox>
+            <DoubleSlider ShowMedian DoubleCol Indent DisableTooltip Id="runnabilityDoubleSlider" IsEnabled="@(Flags.UnrunnableShuffle != false)" MinValue="0" MaxValue="100" Step="4" @bind-ValuesRange="(Flags.UnrunnablesLow,Flags.UnrunnablesHigh)">Count:</DoubleSlider>
```

Comment:
the double slider shows the median from a nonlinear scaling function that is normally used with the control. I think there is an option to make it linear, look for the EXP Chest Convertsion percent.